src/script/page/LeftSidebar/panels/Conversations/ConversationTabs/ConversationTabs.styles.ts

```diff
@@ -23,10 +23,17 @@ export const footerDisclaimer: CSSObject = {
   display: 'grid',
   gap: '10px',
   gridTemplateColumns: '1fr auto',
+  marginBottom: '4px',
   padding: '8px',
+
+  '.footer-disclaimer-tooltip': {
+    alignItems: 'center',
+    display: 'flex',
+  },
```

Comment:
Is there any specific reason to add these style under the class, not as CSS object?